curiosrc/ffi/sdr_funcs.go

```diff
@@ -489,7 +489,24 @@ func (sb *SealCalls) MoveStorage(ctx context.Context, sector storiface.SectorRef
 
 	toMove := storiface.FTCache | storiface.FTSealed | moveUnsealed
 
-	err := sb.sectors.storage.MoveStorage(ctx, sector, toMove)
+	var opts []storiface.AcquireOption
+	if taskID != nil {
+		resv, ok := sb.sectors.storageReservations.Load(*taskID)
+		if ok {
```

Comment:
what if !ok ?

Reply:
Then everything still works, just without the storage reservation - which iiuc can only happen on restart, and the reservation will get restored as soon as the task starts (MoveStorage will reserve space internally if not provided with a reservation).

Reply:
A comment here would be helpful